File: Source/OregonTilth.Web/src/app/shared/models/forms/harvest-post-harvest-standard-times/harvest-post-harvest-standard-time-create-dto.ts
import { FieldLaborActivityDto } from "../../generated/field-labor-activity-dto";
import { FieldUnitTypeDto } from "../../generated/field-unit-type-dto";
import { LaborTypeDto } from "../../generated/labor-type-dto";
import { MachineryDto } from "../../generated/machinery-dto";

export class HarvestPostHarvestStandardTimeCreateDto {
    WorkbookID: number;
    CropID: number;
    CropUnitID: number;
    HarvestTypeID: number;
    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}

File: OregonTilth.Models/DataTransferObjects/Workbook/Forms/HarvestPostHarvestStandardTime/HarvestPostHarvestStandardTimeCreateDto.cs
using System.ComponentModel.DataAnnotations;

namespace OregonTilth.Models.DataTransferObjects
{
    public class HarvestPostHarvestStandardTimeCreateDto
    {
        [Required]
        public int WorkbookID { get; set; }
        [Required]
        public int CropID { get; set; }
        [Required]
        public int CropUnitID { get; set; }
        [Required]
        public int HarvestTypeID { get; set; }
    }
}